### PR TITLE
Fix `label`

### PR DESCRIPTION
**Pull request by vovaf709**

### CODE REVIEW

File: benchmarks/benchmark_zoom.py [hunk 1]
```diff
@@ -26,12 +26,39 @@ class ZoomSuite:
 
     @discard_arg(1)
     def setup(self, dtype):
-        self.image = np.random.randn(256, 256, 256).astype(dtype)
+        self.image_1d = np.random.randn(2**24).astype(dtype)
+        self.image_2d = np.random.randn(4096, 4096).astype(dtype)
+        self.image_3d = np.random.randn(256, 256, 256).astype(dtype)
+        self.image_4d = np.random.randn(64, 64, 64, 64).astype(dtype)
```
Comment (maxme1) [inline benchmarks/benchmark_zoom.py:32]: why not use a separate parameter "ndim" for this?

File: imops/measure.py [hunk 2]
```diff
@@ -120,7 +120,7 @@ def label(
         res.append(('labels', np.array(range(1, num_components + 1))))
     if return_sizes:
         _, sizes = unique(labeled_image, return_counts=True)
-        res.append(('sizes', sizes[1:]))
+        res.append(('sizes', sizes[1:] if 0 in labeled_image else sizes))
```
Comment (maxme1) [inline imops/measure.py:123]: isn't this check super slow?

File: imops/measure.py [hunk 3]
```diff
@@ -120,7 +120,7 @@ def label(
         res.append(('labels', np.array(range(1, num_components + 1))))
```
Comment (maxme1) [inline imops/measure.py:120]: is this true in case when there is no background?

File: imops/measure.py (hunk 2 above)
Comment (vovaf709) [reply imops/measure.py:123]: No, e.g. for array of size `(512, 512, 512)` labeling took ~7 sec. while `0 in labeled_image` took ~0.1 sec.

File: imops/measure.py (hunk 3 above)
Comment (vovaf709) [reply imops/measure.py:120]: Yeah, labels are started with 1. There are also check in `test_measure.test_no_background`, `test_measure.test_no_background2`

File: benchmarks/benchmark_zoom.py (hunk 1 above)
Comment (vovaf709) [reply benchmarks/benchmark_zoom.py]: :fair_enough:

File: imops/measure.py (hunk 2 above)
Comment (maxme1) [reply imops/measure.py:123]: fair enough

**Review by maxme1 — APPROVED** [review]

**Pull request merged**